server/routes/bitbucket/bitbucket.py

```
@@ -270,11 +269,27 @@ def bitbucket_status(user_id):
     try:
-        from utils.secrets.secret_ref_utils import delete_user_secret
-
         # Delete both bitbucket credentials and workspace selection
         delete_user_secret(user_id, "bitbucket")
-        delete_user_secret(user_id, "bitbucket_workspace_selection")
+
+        # Also clean up legacy vault entry if it exists
+        try:
+            delete_user_secret(user_id, "bitbucket_workspace_selection")
+        except Exception:
+            pass
```

Ruff (0.15.15)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)